.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,10 +6,11 @@ Briefly describe the changes proposed in this PR:
 <!-- short description of your change goes here -->
 
 ---- 
-PR Author Checklist: 
```

Comment:
Could you add the maven command for formatting somewhere on this template as a comment?

Reply:
> > Other than my comment. Looks good.
> 
> Sorry, which comment is that?

GitHub had made it a draft comment. Sorry for the confusion. Fixed it now.